src/components/interfaces/HMI_API.xml

```diff
@@ -503,6 +503,45 @@
     <element name="turnText"/>
 </enum>
 
+<enum name="TextFieldType">
+  <element name="mediaTitle">
+    <description>The data in this field contains the title of the currently playing audio track.</description>
+  </element>
+  <element name="mediaArtist">
+    <description>The data in this field contains the artist or creator of the currently playing audio track.</description>
+  </element>
+  <element name="mediaAlbum">
+    <description>The data in this field contains the album title of the currently playing audio track.</description>
+  </element>
+  <element name="mediaYear">
+    <description>The data in this field contains the creation year of the currently playing audio track.</description>
+  </element>
+  <element name="mediaGenre">
+    <description>The data in this field contains the genre of the currently playing audio track.</description>
+  </element>
+  <element name="mediaStation">
+    <description>The data in this field contains the name of the current source for the media.</description>
+  </element>
+  <element name="rating">
+    <description>The data in this field is a rating.</description>
+  </element>
+  <element name="currentTemperature">
+    <description>The data in this field is the current temperature.</description>
+  </element>
+  <element name="maximumTemperature">
+    <description>The data in this field is the maximum temperature for the day.</description>
+  </element>
+  <element name="minimumTemperature">
+    <description>The data in this field is the minimum temperature for the day.</description>
+  </element>
+  <element name="weatherTerm">
+    <description>The data in this field describes the current weather (ex. cloudy, clear, etc.).</description>
+  </element>
+  <element name="humidity">
+    <description>The data in this field describes the current humidity value.</description>
+  </element>
```

Comment:
@mfoedisch Yes, this requires a PR to the proposal, because the [agreed upon proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0073-Adding-Metadata-Types.md#additions-to-hmi_api) still includes this element. This PR is supposed to be an implementation of that proposal exactly. The proposal NEEDS to be changed for this value to be removed.